cmd/bridge/main.go

```diff
@@ -107,6 +107,14 @@ func main() {
 	fStatuspageID := fs.String("statuspage-id", "", "Unique ID assigned by statuspage.io page that provides status info.")
 	fDocumentationBaseURL := fs.String("documentation-base-url", "", "The base URL for documentation links.")
 
+	fMonitoringAlermanagerURL := fs.String("monitoring-alermanager-url", "", "Alertmanager monitoring URL.")
+	fMonitoringGrafanaURL := fs.String("monitoring-grafana-url", "", "Grafana monitoring URL.")
+	fMonitoringPrometheusURL := fs.String("monitoring-prometheus-url", "", "Prometheus monitoring URL.")
+	fMonitoringThanosURL := fs.String("monitoring-thanos-url", "", "Thanos monitoring URL.")
+
+	fLoggingKibanaAppURL := fs.String("logging-kibana-app-url", "", "KibanaApp logging URL.")
+	fLoggingKibanaInfraAppURL := fs.String("logging-kibana-infra-app-url", "", "KibanaInfraApp logging URL.")
```

Comment:
Do we plan to put both of these URLs in the left nav? Do we need both? If we do, we probably want some kind of RBAC check for the infra URL since not all users have access.

cc @jcantrill

Reply:
Currently we dont need the kibana-infra URL. I added it here since we the logging operator is providing it. Thought that since we are doing this change it cant hurt to add it. If anything I can remove it.